evap/templates/base.html

```diff
@@ -127,8 +127,10 @@
         {% block rawcontent %}
             <div class="row">
                 <div class="col-md-12">
-                    {% block content %}
-                    {% endblock %}
+                    <div>
```

Comment:
this change moves the buttons next to the heading into a nicer position. they are now aligned with the text.